test/main.js

```diff
@@ -95,6 +95,14 @@ const createBrowser = async (browser, testCase) => {
 const createMocha = (driver, testCase) => {
   // Create our Mocha instance
   const mocha = new Mocha({
+    reporter: 'mochawesome',
+    reporterOptions: {
+      overwrite: true,
+      reportTitle: 'UI Tests',
+      reportFilename: 'UITestReport',
+      reportDir: './dist/reports/UITestsReport',
+      assetsDir: './dist/reports/UITestsReport',
```

Comment:
What's this for?

Reply:
That's Path to save report assets - js/css files.
 
`reportDir` is only for the json and html files generated.